package.json

```diff
@@ -69,7 +69,7 @@
     "jest-cli": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
     "ol": "^10.4.0",
-    "ol-mapbox-style": "^12.4.1",
+    "ol-mapbox-style": "^12.5.0",
```

Comment:
In this version, `browser` and `module` entries significantly changed. For the moment I would suggest to leave version as is.

Reply:
Top, this might explain why the build fails. Will revert it.